lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -51,7 +51,7 @@ def cmd_run(*args, action: nil)
       jobify = true
     end
 
-    rhosts = datastore['RHOSTS']
+    rhosts = mod.datastore['RHOSTS']
```

Comment:
It looked like this line was working previously by coincidence, `datastore` was being delegated to the active mod by via the method_missing implementation